### PR TITLE
Use scoring when performing keywords tagging

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
Update of tagger version needed, also depends on https://github.com/fabric8-analytics/fabric8-analytics-tagger/pull/23 that needs to be merged first. Do not merge now please.

**Comment by fridex** [comment]
[test this]

**Comment by fridex** [comment]
[test this]

File: f8a_worker/workers/keywords_tagging.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ class KeywordsTaggingTask(BaseTask):
         'lemmatize': True,
         'stemmer': 'EnglishStemmer',
         'ngram_size': None
+        'scorer': 'RelativeUsage'
```
Comment (jpopelka) [inline f8a_worker/workers/keywords_tagging.py]: missing comma after `None`

Comment (fridex) [inline f8a_worker/workers/keywords_tagging.py]: Thanks! Wrong rebase. Should be fixed now.

**Pull request merged**